packages/frontend/src/map-contents/Map/MapSLO.tsx

```diff
@@ -0,0 +1,10 @@
+import { MAP_URL } from '../../constants/ArcGIS';
+import './MapSLO.scss'
+
+export const MapSLO = () => {
+  return (
+    <div className="embed-container">
+      <iframe width="900" height="700" scrolling="no" title="10000trees2" src={MAP_URL}></iframe>
```

Comment:
No space in title.

Also, when there isn't a child element, it's typical to use `/>` to close the tag in the opening one.